.github/workflows/vulnerability_scanning.yml

```diff
@@ -43,9 +59,30 @@ jobs:
           repository: Wiredcraft/github-actions
           path: ./trivyignore
           ref: trivy-whitelist
+                
+      - name: download docker image from minio
+        if: env.MINIO_CN_ENDPOINTS != null && env.MINIO_CN_ACCESS_KEY_ID != null && env.MINIO_CN_SECRET_ACCESS_KEY != null && env.IMG_TAR_NAME != null
```

Comment:
we need to use `env.MINIO_CN_ENDPOINTS != null && env.MINIO_CN_ACCESS_KEY_ID != null && env.MINIO_CN_SECRET_ACCESS_KEY != null` multiple times. I suggest to make it as a var. similar like what you did for the `calculatedSha`